src/graphql/home/home.graphql

```diff
@@ -147,7 +147,7 @@ query home(
     wallet: "ALphA7iWKMUi8owfbSKFm2i3BxG6LbasYYXt8sP85Upz"
     limit: $feedEventsLimit
     offset: 0
-    excludeTypes: ["follow"]
+    excludeTypes: ["follow", "mint"]
```

Comment:
Removes chance of one user clogging up the feed with mint events in the Hero section